etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/database/PostgresDaoTest.groovy

```diff
@@ -200,7 +203,7 @@ class PostgresDaoTest extends Specification {
         mockPreparedStatement.executeQuery() >> mockResultSet
         mockResultSet.next() >> true
         mockResultSet.getTimestamp("time_received") >> null
-
```

Comment:
In a new test similar to this one, can you test that when the database returns `null` for the `delivery_status`, that it is converted to `PENDING`?  This comment, of course, depends on the answer of the comment asking about whether defaulting to `PENDING` is right or not.

Reply:
Not needed anymore since we moved the logic to the `PartnerMetadata`.